README.md

```diff
@@ -1,2 +1,59 @@
-# aghtetnyein.github.io
-My first portfolio in Microverse deployed with Github Pages
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Sa Aung Htet Nyein's Microverse Portfolio
+
+> My personal portfolio and the first project of Module 1 at Microverse.
+
+## Live Demo
+
+[To my Portfolio Live Demo](https://aghtetnyein.github.io)
+
```

Comment:
- [ ] Remember that this link is supposed to display your actual portfolio page.
### Your display
![image](https://user-images.githubusercontent.com/98453979/187141884-6b496bcf-fb5b-4154-aa15-1618028265e1.png)

- [ ] Would you please follow these [steps](https://www.freecodecamp.org/news/publish-your-website-netlify-github/) to deploy your website

Reply:
Thanks, much to you @frank1738 
I think the actual portfolio page will appear after I merge this branch to the main and so should I merge it to view the live Demo?

Reply:
- [ ] @aghtetnyein.  You can deploy the current branch. you don't need to merge